### PR TITLE
RoW/DS Update

### PR DESCRIPTION
**Pull request by jofwu**
Just wanting to see how these events look on the timeline...

### CODE REVIEW

File: translations/en/events/taravangians-treachery.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Taravangian's treachery
+Taravangian meets with the inner circle of the Diagram and the group disbands.
```
Comment (lianzhao) [inline translations/en/events/taravangians-treachery.md:2]: This summary duplicated with `diagram-disbands.md` @jofwu

File: src/lang/es-ES.lang.json [hunk 2]
```diff
@@ -151,7 +156,51 @@
...
+                "distance": "Distance"
+            },
+            "coordinates": {
+                "full": "{latitude}, {longitude}",
+                "north": "{degrees}°ᴿ N",
+                "south": "{degrees}°ᴿ S",
+                "east": "{degrees}°ᴿ E",
+                "west": "{degrees}°ᴿ W"
+            },
+            "unit": "imperial",
```
Comment (lianzhao) [inline src/lang/es-ES.lang.json:178]: Hello @PaleoCrafter , I'm trying to translate this into Chinese. May I ask what's this `unit` used for? I searched in source code with keyword `measurement.unit`, but found nothing matched.

Comment (PaleoCrafter) [reply src/lang/es-ES.lang.json]: Hi @lianzhao, good catch! That was left over from when I only had the measurements show one unit of distance (either miles or kilometers). But we're displaying both now so it's not used anymore. `measurement.distance` is the relevant thing now :)

Comment (lianzhao) [reply src/lang/es-ES.lang.json]: Got it. I'll remove this line from `zh.lang.json` as well. Thank you @PaleoCrafter .

File: translations/en/events/taravangians-treachery.md (hunk 1 above)
Comment (jofwu) [reply translations/en/events/taravangians-treachery.md]: @lianzhao I've updated this one! Thanks.

**Comment by lianzhao** [comment]
> 

@jofwu I've opened PR #130 to sync with en.

**Pull request merged**